README.md

```diff
@@ -1,61 +1,94 @@
 # cDMRG
 
-  cDMRG enables the application of the Density-Matrix Renormalization Group (DMRG) to continuous quantum systems by spatial partitioning into segments with continuous basis functions. It is distinct from cMPS. The code is written for bosons with contact interactions trapped in a box with a sinusoidal potential.
+cDMRG enables the application of the Density-Matrix Renormalization Group (DMRG) to continuous quantum
+systems by spatial partitioning into segments with continuous basis functions.
+It is distinct from cMPS.
+The code is written for bosons with contact interactions trapped in a box with a sinusoidal potential.
 
+## Installation
 
-## Modules
+In order to get this code running on your machine follow these steps.
 
-  cDMRG consists of two main modules: a DMRG module using the ITensor library and an input module written in Mathematica.
+### Prerequisites
 
+- We've tested this package on Unix only, however running on Win10 should be straighforward
+  with the build tools.
+- [Install and build ITensor 3 (C++ version)](https://itensor.org/)
+- [Install Mathematica ???](https://www.wolfram.com/mathematica/)
 
-### DMRG Module
+### First steps
+
+- Clone this repository ANYWHERE OR SOME SPECIAL LOCATION WITH RESPECT TO ITENSOR DIR???
```

Comment:
The current make file has DIR=../../itensor, so subsubdirectory. But one can just change this as needed.